runbotics/runbotics-orchestrator-ui/src/src-app/components/cron/converter.ts

```diff
@@ -56,21 +60,24 @@ export function setValuesFromCronString(
 
     if (!error) {
         // Shortcuts management
-        if (shortcuts && (shortcuts === true || shortcuts.includes(newCronString as any))) {
+        if (
+            shortcuts &&
+            (shortcuts === true || shortcuts.includes(newCronString as any))
```

Comment:
why is this as any?

Reply:
I believe its because shortcuts is an array of more precise type than a string but it shouldn't be any for sure

Reply:
and created a typeguard